sys/include/net/ng_pktdump.h

```diff
@@ -39,7 +39,7 @@ extern "C" {
  * @brief   Priority of the pktdump thread
  */
 #ifndef NG_PKTDUMP_PRIO
-#define NG_PKTDUMP_PRIO                 (PRIORITY_MIN - 1)
+#define NG_PKTDUMP_PRIO                 (PRIORITY_MAIN - 1)
```

Comment:
Does a higher priority hurt? The lower one breaks all native shell tests that use getchar (I think), like the tapnet test you wrote.